ibm_was/datadog_checks/ibm_was/data/conf.yaml.example

```diff
@@ -68,3 +68,20 @@ instances:
   #  tags:
   #    - <KEY_1>:<VALUE_1>
   #    - <KEY_2>:<VALUE_2>
+
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel - mandatory - Set port if type is tcp or udp. Set path if type is file and channel if windows_event
+## service - mandatory - Name of the service owning the log
+## source  - mandatory - Attribute that defines which integration is sending the logs
+## sourcecategory - optional - Multiple value attribute. Can be used to refine the source attribtue
+## tags: - optional - Add tags to each logs collected
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+
+#logs:
+#  - type: file
+#    path: opt/IBM/WebSphere/AppServer/profiles/profile/logs/server1/*.log
```

Comment:
should `profile` be `InfoSphere` for the default value of a standalone install, like `server1`

Reply:
Let's put `InfoSphere` by default.